src/Bookings.js

```diff
@@ -1,18 +1,45 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    let filteredBookings = [];
+    bookings.forEach(booking => {
+      if (
+        booking.firstName.toLowerCase().includes(searchVal.toLowerCase()) ||
+        booking.surname.toLowerCase().includes(searchVal.toLowerCase())
+      ) {
+        filteredBookings.push(booking);
+      }
+    });
+    setBookings(filteredBookings);
   };
 
+  const [bookings, setBookings] = useState(0);
+  const [profile, setProfile] = useState(0);
+
+  useEffect(() => {
+    console.log("use effect...");
+    // fetch("https://cyf-react.glitch.me/delayed")
+    fetch("https://cyf-react.glitch.me/error")
```

Comment:
assuming you must be testing out the error response and that's why this is in here :)